chapters/packages.tex

```diff
@@ -321,3 +321,184 @@ \section{External Resources}\label{external-resources}
 \filename{modelica://Modelica/Mechanics/C.jpg} is illegal.
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
+
+\section{Multilingual Descriptions}\label{multilingual-descriptions}
+
+\begin{nonnormative}
+Descriptive texts in a model or library are usually formulated in English.
+This section describes how a tool can present the library in another language.
+Translated Modelica text is provided by external files, so that no modification of the original Modelica text is required.
+\end{nonnormative}
+
+The texts in following Modelica constructs should be translated:
+\begin{itemize}
+\item description strings of component declarations and classes
+\item strings in the following annotations:
+\begin{itemize}
+\item \lstinline!Text.string!, \lstinline!Text.textString!
+\item \lstinline!missingInnerMessage!, \lstinline!obsolete!, \lstinline!unassignedMessage!
+\item \lstinline!Dialog.group!, \lstinline!Dialog.tab!
+\item \lstinline!Dialog.loadSelector.caption!, \lstinline!Dialog.loadSelector.filter!, \lstinline!Dialog.saveSelector.caption!, \lstinline!Dialog.saveSelector.filter!
```

Comment:
Line breaks do not look that good. Maybe have two sperate items for loadSelector and saveselector.

![grafik](https://user-images.githubusercontent.com/14896695/204051155-bef3732f-19fa-47cf-a149-b028e75b86b7.png)

Reply:
To avoid separating the closely related constructs into separate items we can introduce a PDF-only line-break:
```suggestion
\item
  \lstinline!Dialog.loadSelector.caption!, \lstinline!Dialog.loadSelector.filter!,
  \ifpdf\\\else\fi% Avoid comma ending up at beginning of wrapped line in PDF output.
  \lstinline!Dialog.saveSelector.caption!, \lstinline!Dialog.saveSelector.filter!
```